src/components/Map.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  MapContainer,
+  TileLayer,
+  Marker,
+  Popup,
+} from 'react-leaflet';
+import myIcon from '../constants/icon';
+import { observable$, filterLocation } from '../utils/mapAPI';
+
+const useStyles = makeStyles(() => ({
+  map: {
+    height: '300px',
+  },
+}));
+
+const Map = () => {
+  const [elements, setElements] = useState([]);
+  const classes = useStyles();
+  useEffect(() => {
+    observable$.subscribe((res) => {
+      setElements(filterLocation(res));
+    });
+  }, []);
+
+  return (
+    <MapContainer
+      center={[51.11535749698708, 17.034426774577256]}
+      zoom={7}
+      className={classes.map}
+    >
+      <TileLayer
+        attribution='&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
+        url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
+      />
+      {elements.length && elements.map(({ location, id }) => (
+        <>
+          {location.length && (
+          <Marker position={location.split(',')} icon={myIcon} key={id}>
+            <Popup>
+              A pretty CSS3 popup.
+              <br />
+              Easily customizable.

```

Comment:
wyświetl tu nazwę z jsona

Reply:
Tutaj on chciał żeby był link do danego posta

Reply:
no to akurat nazwa która będzie linkiem, możesz już zrobić w sumie ```<Link to=`/wyprawy/${id}`>{label}</Link>```

Reply:
trzeba będzie w php zmienić tą funkcję visite-places.php, ona zwraca coś takiego:
'id' => $post->ID,
'location' => get_field('lokalizacja', $post->ID)

nie wiem czy coś takiego dodać:
'title' => get_field('title', $post->ID)